src/mercedestrenz/data.py

```diff
@@ -52,12 +55,73 @@ def listing_search(data, budget=[0, np.Inf], model = "any", sort_feature = "odom
     
     Examples
     --------
-    >>> from mercedestrenz.mercedestrenz import listing_search
-    >>> result = listing_search(data, budget=[2000, 20000], model = "any", sort_feature = "odometer", ascending = True)
-    >>> result.head()
-    #to be completed with exmaple output
+    >>> # search listings within a price range
+    >>> listing_search(data, budget=[2000, 20000], model = "any", sort_feature = "odometer_mi", ascending = True)
+    >>> # search listings for a specific model and below a maximum price
+    >>> listing_search(data, budget=20000, model = "glk", sort_feature = "odometer_mi", ascending = True)
```

Comment:
- We might need to provide a list of the possible model types. In this example, technically "glk" is not one of the available model types in our data. The list of possible models is:
    - `['other', 'm-class', 's-class', 'cl-class', 'e-class', 'c-class',
       'gl-class', 'sl-class', 'metris', 'b-class', 'amg', 'sprinter',
       'a-class', 'r-class', 'd-class']`
- I suggest adding this list to the documentation under the `model` parameter. I'm not able to add a comment to that specific section since there weren't any new additions/deletions there.
```suggestion
    >>> listing_search(data, budget=20000, model = "gl-class", sort_feature = "odometer_mi", ascending = True)
```